src/authentication/AuthenticationSection.tsx

```diff
@@ -73,9 +73,9 @@ export default function AuthenticationSection() {
       throw new Error(t("common:notFound"));
     }
 
-    const defaultFlows = Object.entries(realmRep)
-      .filter((entry) => REALM_FLOWS.includes(entry[0]))
-      .map((entry) => entry[1]);
+    const defaultFlows = Object.entries(realmRep).filter((entry) =>
+      REALM_FLOWS.includes(entry[0])
+    );
```

Comment:
This value could also be destructured.
```suggestion
    const defaultFlows = Object.entries(realmRep).filter(([key]) =>
      REALM_FLOWS.includes(key)
    );
```